17:24	Saegrimr	Then i'm going to start banning nonantags who buy it.

Iamgoofball opened the pull request:
:cl: Saegrimr
del: Removed all shuttles that inconvenience or risk any injury to players, to prevent captains from being banned for purchasing/getting paid to use shuttles that are sub-par, kill, or ruin the fun of the majority of players.
/:cl:

http://pastebin.com/kLhneVL6 full logs for future reference on rulings related to shuttles and captains

![chrome_2016-11-18_17-53-25](https://cloud.githubusercontent.com/assets/4081722/20452140/014dc700-adb8-11e6-9c72-c5dec635b9ae.png)

Iamgoofball committed:
17:24	Saegrimr	Then i'm going to start banning nonantags who buy it.

I have removed all shuttles that inconvenience or risk any injury to players, to prevent captains from being banned for purchasing/getting paid to use shuttles that are sub-par, kill, or ruin the fun of the majority of players.

ExcessiveUseOfCobblestone commented:
why not make the meme shuttles cost a lot instead of give you points for grieffing the crew?

Iamgoofball commented:
you'll still get banned for griefing the crew

and they gave credits so that you'd have an incentive to actually get them

Kraseo commented:
Repeat after me:

# **H U G B O X**

Iamgoofball commented:
@Kraseo it's bannable so

Iamgoofball commented on code/datums/shuttles.dm:
```diff
@@ -178,20 +138,6 @@
-	admin_notes = "If the crew can solve the puzzle, they will wake the \
-	wabbajack statue. It will likely not end well. There's a reason it's \
-	boarded up. Maybe they should have just left it alone."
-	credit_cost = 15000
```
dangerous and can get players easily griefed

Iamgoofball commented on code/datums/shuttles.dm:
```diff
@@ -159,17 +130,6 @@
-		I cannot tell you; it's space confidential. \
-		Aw, come space on. Why not? \
-		No, I can't. Anyway, how is your space roleplay life?"
-	credit_cost = -5000
```
small amount of space and spawns with a gun you can grief players with which is what usually happens

Iamgoofball commented on code/datums/shuttles.dm:
```diff
@@ -99,21 +85,6 @@
-		crates!"
-	admin_notes = "Due to origin as a solo piloted secure vessel, has an \
-		active GPS onboard labeled STV5."
-	credit_cost = -7500
```
spawns with loads of guns, cramped space, lots of grief potential

Iamgoofball commented on code/datums/shuttles.dm:
```diff
@@ -57,20 +57,6 @@
-	name = "An Asteroid With Engines Strapped To It"
-	description = "A hollowed out asteroid with engines strapped to it. Due to its size and difficulty in steering it, this shuttle may damage the docking area."
-	admin_notes = "This shuttle will likely crush escape, killing anyone there."
-	credit_cost = -5000
```
press button receive ban for non-antag bombing all of escape

ExcessiveUseOfCobblestone commented:
> and they gave credits so that you'd have an incentive to actually get them

That's the problem, they give you points to antagonize the crew [as nonantag] FNR.

Iamgoofball commented:
removing the incentive isn't going to stop them from griefing the crew

Kraseo commented:
If people can't get out of the way with CLEARLY MARKED PLACES THE SHUTTLE IS GOING TO CRASH INTO they don't deserve living tbh.

Saegrimr commented:
Good job only posting part of the IRC logs you liked you snarky cunt.

http://pastebin.com/kLhneVL6

Iamgoofball commented:
i was too lazy to screencap it all, just grabbed the important bit

will add to OP, thanks bae

Saegrimr commented:
Good. If you're gonna try and make me look bad, use the whole thing so I don't feel like I wasted an entire 20 minutes of arguing for nothing.

Iamgoofball commented:
👌

Iamgoofball commented:
don't see it as a waste btw, i'm saving you time from having to deal with appeals over shuttle purchases by doing that

KorPhaeron commented:
>they give an incentive to do something for no reason

???

Iamgoofball commented:
an extra 5000 cargo points can buy a lot of meme shit kor

B00tz commented:
Why not have it so when a dangerous shuttle is bought it lists the dangers of it so people know to stand farther and such.

ExcessiveUseOfCobblestone commented:
yeah sorry im dumb.

They give an incentive to grief the crew as [usually] a nonantag. Just that

Iamgoofball committed:
Removed snappop to prevent shuttle grief via chasms

easy to permakill someone with this shuttle, it's a pretty good pick for traitors looking to shuttle grief

Iamgoofball commented on code/datums/shuttles.dm:
```diff
@@ -83,37 +69,6 @@
-	admin_notes = "Brig is replaced by anchored greentext book surrounded by \
-		lavaland chasms, stationside door has been removed to prevent \
-		accidental dropping. No brig."
-	credit_cost = 8000
```
easy to permakill people via shuttle grief by opening the all access door and throwing people into chasm tiles

KorPhaeron commented:
Should buying destructive shuttles (like the giant asteroid) result in a ban for the captain?
Yes, it kills too many people: 2
No, it's funny and people can move out of the way: 37
I don't know what you're talking about or care very much either: 4
Did not vote: 31

Saegrimr commented:
>Why not have it so when a dangerous shuttle is bought it lists the dangers of it so people know to stand farther and such.

This was offered as a suggestion, which I replied that the

>CLEARLY MARKED PLACES THE SHUTTLE IS GOING TO CRASH INTO they don't deserve living tbh.

Aren't noticable on the station tiles. If the area is packed, nobody is going to be able to push eachother out of the way in time.

It's good that goof is being a bigger memelord than I am just to get his own PR shut down to try and prove a point.

ExcessiveUseOfCobblestone commented:
Right so if we don't want to ban them [which I have no problem with] then make it more difficult to obtain the "lol kill the crew" shuttles by making them cost more [and certainly not give you points].

Niknakflak commented:
Just close this salt fueled bullshit

Incomptinence commented:
@Saegrimr 

Seems like everything was find before your back seat headmin/headcodering.

At most requiring a new sprite for the indicators.

@Niknakflak 

Typical niknak

Saegrimr commented:
>At most requiring a new sprite for the indicators.

Taken from the round just minutes ago on Sybil.
Indicators as they arrive, barely noticable.
http://i.imgur.com/WYVRJ9O.png

Max brightness, this is however too late as the shuttle is arriving right now.
http://i.imgur.com/d3fHa1R.png

Destroyed everything up to the science entrance doors, R&D just plain gone.
http://i.imgur.com/MTYeYNC.png

Not even electrical maint is safe.
http://i.imgur.com/pfa9Tsl.png

[19:47:37] <Saegrimr> You say that like we haven't banned people for bombing the shuttle or just shuttle grief in general.

Every round is this shitty meme of a shuttle, 5000 credits is practically fucking nothing to justify it with.
To quote the fucking MotD.

" Try not to run this into the ground."

Bawhoppen commented:
goof being trash again

Kraseo commented:
"barely noticeable" if you're as blind as a bat maybe

Niknakflak commented:
@Incomptinence 
It's one thing to have a debate about reverting the asteroid shuttle.  It's another for goofball to go in and remove a shit ton of other shuttles that don't really have any problems because he makes up some bullshit "grief potential".

Take your shitty bias elsewhere.

Saegrimr commented:
@Kraseo 

![Nice contrast](http://i.imgur.com/rztBluu.png)

Seriously?

KorPhaeron commented:
![seemsfine](http://i.imgur.com/ecxJ0J4.png)

??

Saegrimr commented:
@KorPhaeron 
>Max brightness, this is however too late as the shuttle is arriving right now.

Yes thank you for showcasing the next screenshot I had.

Saegrimr commented:
While we're also on the topic of pasting log snips and goofball not understanding the difference between gibbing an entire area full of people and having to sit on someone's lap.

(iamgoofball) and would you apply this "fun ruining" issue to any of the other shuttles that inconvenience crew
(Saegrimr) Wasn't there one that was literally just plating and no walls and airless? That's probably also a bad idea.
(iamgoofball) no but there's stuff like small cramped shuttles, a shuttle you can only enter if you have 500 bucks on you, etc.
(Saegrimr) That one is hilarious but it still has a poverty section for people to get on.
(iamgoofball) yes but it's small and cramped and most the station can't get on it
(iamgoofball) therefore, it's ruining the fun of the people who can't get on
(Saegrimr) b-but the captain's fun
(iamgoofball) there you go
(Saegrimr) Do you not see how gibbing the escape hallway and everything around it is worse than cramped spaced?
(Saegrimr) Besides i've seen 20 people fit on a pod with laying down. Up until someone opened the door anyway.
(iamgoofball) i'm removing the asteroid shuttle now
(iamgoofball) i just don't think dying in a 30 minute round based game is a bad thing when it gives you access to the grief lottery on steroids within the next 5-10 minutes
(Saegrimr) You say that like we haven't banned people for bombing the shuttle or just shuttle grief in general.
(iamgoofball) i don't think it matters all that much either but my opinion doesn't matter

KorPhaeron commented:
I've personally never had any trouble moving out of the way of the ripples, even for the asteroid shuttle.

This is going away from the broader argument of whether the captain should have round altering powers/authority though.

Kraseo commented:
@Saegrimr I mean fuck there's a slight flash before the circles, and even then it gradually gets brighter so I dunno how you can miss it.

KorPhaeron commented:
Anyway I think it's a kneejerk reaction to ban people for overusing it when it's both less than a week old and overwhelmingly supported by players right now. There are very few things in the game less controversial at this time.

Rules are to make sure people have a fun time, and right now people are still having a fun time with it.

I'll fuck off now and let the headmins decide

Saegrimr commented:
@KorPhaeron 
Shame about the whole "Try not to run this into the ground." thing when its bought immediately every round completely irrelevant of cargo's status.

@Kraseo 
Why should people who don't get to see the message about HUGE ASTEROID at roundstart have to expect the escape shuttle to show up on top of them and kill them? Just edit the map and make a larger docking port then if this is going to be the norm.

Or they can stop buying the station destroying shuttle every round I dunno man.

KorPhaeron commented:
@Saegrimr 

Happens with literally every new feature

The asteroid shuttle docks fine at this size by the way, the port gets moved into the station when you buy it to cause that crash

Saegrimr commented:
So why is there a feature to destroy a large portion of the station in the first place? And why is it okay for nonantags to use it but not okay for nonantags to bomb it?

ExcessiveUseOfCobblestone commented:
that doesn't change the fact you get paid for griefing the crew.

HankMoodyExperience commented:
**H U G B O X S T A T I O N**

SgtxRootbeer commented:
@saegrimr
It crashes because of lore reasons...
You know, it's a giant rock with engines strapped to it, it isn't nimble enough to actually get to the station without swapping paint...
(Why it docks perfectly at centcomm, idk. Maybe centcomm has hyperspatial tugboats to assist.)

Saegrimr commented:
2/3 Headmin confirmation (Third unreachable)

- OK for captains to buy the shuttle (bwoink if done too often, ban if they keep doing it)
- OK  for people who break in and steal the spare to buy the shuttle (bwoink if done too often, ban if they keep doing it)

Reminder that:
- I still haven't banned anybody for this
- I still haven't even warned anybody for this
- I still haven't even told someone to chill out on ordering these
- This entire drama bomb is Goofball starting more bullshit and making a meme PR instead of going with what he first offered which is a shuttle notification.
- Remove Goofball

I'd like to thank everybody involved, myself included, for falling for goof bait when literally nothing happened aside from him trying to remove a bunch of shuttles like a kid kicking his own lego fort down.

kevinz000 commented:
Goddamnit Goofball overexaggerating shit.

optimumtact commented:
This should be a configurable list for server admins by the way.

scoopscoop commented:
I really don't mind the new shuttles, it's just that there is no downside for picking a terrible shuttle.

Yeah cargo losing points doesn't count, captains don't give a shit and cargo can't do anything about it.

CPTANT commented:
Holy mother of salt

Someone *coughSaegrimr* *coughSaltball* should really read rule 10 again.


**10. Losing is part of the game.**
    Your character will frequently die, sometimes without even a possibility of avoiding it. Events will often be out of your control. No matter how good or prepared you are, sometimes you just lose.

Arianya commented:
"Someone" _cough CPTANT cough Idiotball_ should really read Rule 1


**1. Don't be a dick.**
We're all here to have a good time, supposedly. Going out of your way to seriously negatively impact or end the round for someone with little IC justification is against the rules. Legitimate conflicts where people get upset do happen however, these conflicts should escalate properly and retribution must be proportionate." 

And hey, we even have a spicy precedent to remind people about how specifically that rule is interpreted!

**1.** Random murders are not acceptable nor is the killing of other players for poor or little reasoning such as ‘My character is insane’. Each unjustified kill is normally met with one 24 ban.

jughu commented:
>im gonna ban people who buy shitty shuttles

>they better gamble the asteroid shuttle and hope that nobody dies

>asteroid shuttle litteraly shows the shuttle area docking thing before it lands

CPTANT commented:
ohw no the escape shuttle is slightly more dangerous, such a dick move!

Arianya commented:
Yes, obviously that was my point and not the fact that it can and will 
murder people and thus breaks rule 1.


10/10 for reading comprehension


On 19/11/2016 18:00, CPTANT wrote:
>
> ohw no the escape shuttle is slightly more dangerous, such a dick move!
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub 
> <https://github.com/tgstation/tgstation/pull/21627#issuecomment-261729189>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/AWUa5BF0eySLzGYH3OfG_1h0LLLe5e2fks5q_zlQgaJpZM4K3H-M>.
>

jughu commented:
The shuttle litteraly shows where it lands by marking the fucking area with blue circles if people die by this they are either stupid or new guys

Bawhoppen commented:
close and lock thread for ultimate win

Crushtoe commented:
@Saegrimr Just want to assure you that it'll slow down. Peopel have already gotten bored of rushing the vault to drain it every round after 3 or 4 days, pretty soon they'll move on past the rock (which doesn't even give the most points.)

Thanks for not banning people while they're just excited to mess around with a shiny new feature and understanding that people have curiosity ❤️

factoryman942 commented:
If it's hard for people to see, just make asteroid shuttle landing zone have a big red overlay like fire alarm for 30 secs before it lands. Or something.

nullbear commented:
Maybe make them only show if you have < x credits. So you can only get when u need to.
